wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearning/ActiveLearningFilmStripCard.vue

```diff
@@ -153,7 +152,7 @@ export default Vue.extend({
          * the current superpixel, we use the right index.
          */
         categoryIndex(label) {
-            return _.map(this.superpixelDecision.categories, (category) => category.label).indexOf(label);
+            return _.map(this.superpixelDecision.labelCategories, (category) => category.label).indexOf(label);
         }
```

Comment:
It is more useful to have this function find the index of a category in the label annotation, since this is the value we'll ultimately need to be setting on the pixelmap.